napalm/eos.py

```diff
@@ -512,3 +513,210 @@ def cli(self, commands = None):
                 raise CommandErrorException(str(cli_output))
 
         return cli_output
+
+    def get_bgp_config(self, group = '', neighbor = ''):
```

Comment:
Have you considered checking TextFSM? Haven't used it but looks promising.

https://github.com/google/textfsm
http://jedelman.com/home/programmatic-access-to-cli-devices-with-textfsm/

I am not implying you should refactor the method, I just wanted to mention it : )